app/src/main/res/layout/app_bar.xml

```diff
@@ -19,7 +19,7 @@
             android:layout_height="match_parent"
             android:layout_margin="10dp"
             android:layout_weight="1"
-            android:background="@color/white"
+            android:background="@color/colorPrimary"
```

Comment:
This parameter should not be changed, as it will give the background the same color as the icon, therefore making it to be invisible